src/main/java/io/jsonwebtoken/BadAudienceJwtException.java

```diff
@@ -0,0 +1,11 @@
+package io.jsonwebtoken;
+
+public class BadAudienceJwtException extends ClaimJwtException {
```

Comment:
hmm... Maybe it should be `ExpectedClaimException` - there was an expectation that wasn't met.

`IncorrectClaimException` is technically correct, but it doesn't match up with the `expect` and `expectIssuer`, etc. methods. Not a big showstopper for me, though - I get stuck on words sometimes ;)